magento/pwa-studio
# [Cypress] Add multi store integration tests (part 1)

## Description

Add integration tests for the multi store feature

## Related Issue







Closes PWA-2470

## Acceptance





@dpatil-magento 

### Verification Stakeholders



Any core developer

### Specification




## Verification Steps

1. Run cypress test under `/src/tests/integrationTests/multiStore/storeSwitching.spec.js`
2. Verify tests pass






#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
